lib/cext/preprocess.rb

```diff
@@ -228,6 +248,40 @@ def read_write_field(struct_var_name, field_name, leaking)
         replacement: MYSQL2_FREE_STATEMENT
       }
     ]
+  },
+  'byebug.c' => {
+    gem: 'byebug',
+    patches: [
+      {
+        match: /\b(static VALUE .*?) = (?:Qfalse|Qnil);/,
+        replacement: '\1;'
+      },
+      {
+        match: /VALUE threads = Qnil;/,
```

Comment:
Is this at the start of a line? In that case an anchor like `^` would be useful so local variables are less likely to be matched.

Reply:
Yes, fixed.